src/openrct2/world/footpath.c

```diff
@@ -513,7 +543,8 @@ void game_command_place_footpath(sint32 *eax, sint32 *ebx, sint32 *ecx, sint32 *
 
 static money32 footpath_place_from_track(sint32 type, sint32 x, sint32 y, sint32 z, sint32 slope, sint32 edges, sint32 flags)
```

Comment:
Should a track design really be able to replace the entrance path?

Reply:
There's no reason why you shouldn't be able too. It's the same type of change for placing path normally.